style.css

```diff
@@ -5,41 +5,61 @@ body{
 
 @media(min-width: 769px){
  div{
+   text-align: center;

```

Comment:
you need to use some SMACSS modularity in your lab so refactor that as much as you can also take a look at the mockups and where your breakpoints should be. otherwise, the styling and page look great!